server/command_create.go

```diff
@@ -12,6 +12,7 @@ import (
 	flag "github.com/spf13/pflag"
 )
 
+var dockerRepoWhitelist = []string{"mattermost/mattermost-enterprise-edition", "mattermost/mm-ee-test", "mattermost/mm-ee-cloud", "mattermost/mm-te", "mattermost/mattermost-team-edition"}
```

Comment:
Just a style request: could we put newlines after each of these items in this list just so the list appears vertically?

Reply:
I'll go ahead and do that Ian thank you for the feedback